plugins/channelrx/noisefigure/noisefiguresink.cpp

```diff
@@ -26,7 +26,6 @@
 #include "noisefiguresink.h"
 
 NoiseFigureSink::NoiseFigureSink(NoiseFigure *noiseFigure) :
-        m_noiseFigure(noiseFigure),
```

Comment:
Remove from constructor parameters if not needed

Reply:
ok